qualtran/bloqs/chemistry/sf/single_factorization_test.py

```diff
@@ -120,8 +120,9 @@ def test_compare_cost_to_openfermion():
         + QR2(num_aux + 1, nprime, bp)[-1]
         + QI2(num_aux + 1, nprime)[-1]
     )
+    delta_relf_missing_ctrl = 1
```

Comment:
relf?

Reply:
relf_on_the_shelf will also disappear once the control issue is fixed.

Reply:
does it mean reflection? wouldn't that be refl?

Reply:
it's a typo yes

Reply:
Done.